cdp_scrapers/instances/seattle.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from ssl import get_protocol_name
+from typing import Dict, List
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+from urllib.error import HTTPError
+import re
+import sys
+import datetime
+
+from cdp_backend.pipeline.ingestion_models import (
+    EventIngestionModel,
+    Body,
+    Session,
+    EventMinutesItem,
+    MinutesItem,
+    Matter,
+    SupportingFile,
+    Person,
+    Vote,
+)
+from cdp_scrapers.legistar_utils import(
```

Comment:
Relative imports should be done with `..` syntax.

In this case:
```python
from ..legistar_utils import get_legistar_events_for_timespan, LEGISTAR_BASE
```

Reply:
41266313603d4851ed0f669f21cc184f9a1bc6b1